sdk/cmake/config.cmake

```diff
@@ -36,7 +36,7 @@ else()
 "Whether to compile for debugging.")
 endif()
 
-if(MSVC)
+if(MSVC AND (NOT USE_CLANG_CL))
```

Comment:
lgtm

Reply:
I'm curious what the reason for this is? Lack of good pdb symbol support? Issues compiling kd64?

Reply:
I just wanted to start with a gcc equivalent of a bootcd, as in without having to attach windbg to proceed further...etc.